curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/adding-a-default-option-in-switch-statements.md

```diff
@@ -31,38 +31,38 @@ switch (num) {
 # --instructions--
 
 Write a switch statement to set `answer` for the following conditions:  
-`"a"` - "apple"  
-`"b"` - "bird"  
-`"c"` - "cat"  
-`default` - "stuff"
+`a` - `apple`  
+`b` - `bird`  
+`c` - `cat`  
+`default` - `stuff`
 
 # --hints--
 
-`switchOfStuff("a")` should have a value of "apple"
+`switchOfStuff("a")` should have a value of `apple`
```

Comment:
Do we want to add "the string " before all of these in these challenge hints?

Reply:
That's a good idea, yeah.